src/lib/deploy.js

```diff
@@ -329,13 +329,11 @@ async function deploy({ alias, yes }) {
     );
     await shutdown();
     return;
-  } else {
-    // Account nonce value is found from the network
-    nonce = Number(accountResponse.data.account.nonce);
   }
 
   let transaction = await step('Build transaction', async () => {
-    addCachedAccount({ publicKey: zkAppAddressBase58, nonce });
+    let Berkeley = Mina.BerkeleyQANet(graphQLEndpoint);
+    Mina.setActiveInstance(Berkeley);
```

Comment:
Does this hardcode deploys to the Berkeley testnet? Wondering if this will cause issues later

Reply:
it doesn't - but the fact that it's called "BerkeleyQANet" is pretty disturbing, I agree. I'll rename it to `Mina.Network` (this was previously agreed on in our team) and update this PR